src/components/ids-data-label/ids-data-label.scss

```
@@ -0,0 +1,50 @@
+@import '../../core/ids-base';
+
+:host([language='fr']) {
```

```suggestion
:host([language='fr']) {
```

I noticed that if using language `fr-FR` or `fr-CA` it doesnt work can we add cases for language = 'fr-FR' and 'fr-CA'?

As those are also valid. And im not sure if just changing here is enough?

---

Could this work?
```suggestion
:host([language*='fr']) {
```